### ISSUE TITLE
Utilize new macros and correct some sb_UT.c merge conflicts

### ISSUE THREAD

**Issue by CDKnightNASA** [nasa/cFE#693]
**Describe the bug**
Doing the merge conflicts for the ic merges, I mis-merged a couple of Joe's changes ( fix #245 ).

**Reporter Info**
Christopher.D.Knight@nasa.gov

**Comment by CDKnightNASA** [comment]
will resolve this once the current IC is merged into master

**Comment by skliper** [comment]
Can it be resolved for this round based on the IC? @astrogeco

**Comment by CDKnightNASA** [comment]
> Can it be resolved for this round based on the IC? @astrogeco

I defer to your best judgement on the process by which my fixes can be merged in.

**Comment by skliper** [comment]
@CDKnightNASA Standby... I'll link a PR under which you'll be able to update/fix (based on current IC).
